### PR TITLE
Appplatform/update custom domain swagger

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by Incarnation-p-lee** [comment]
the diff of some scope looks werid, should be simple change but looks shifted from git default diff layout.

image

**Comment by Incarnation-p-lee** [comment]
@MyronFanQiu 
We have add new field to existed property as below, not sure if it related to breaking change. Any information requires please let me know.

image

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json [hunk 1]
```diff
@@ -953,40 +953,522 @@
...
+            "$ref": "./examples/CustomDomains_Delete.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "AppPlatform"
+        ],
+        "description": "Update custom domain of one lifecycle application.",
+        "operationId": "CustomDomains_Patch",
```
Comment (MyronFanQiu) [inline specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json]: it should be `CustomDomains_Update`

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json [hunk 2]
```diff
@@ -953,40 +953,522 @@
...
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CustomDomains_Get": {
+            "$ref": "./examples/CustomDomains_Get.json"
+          }
+        }
+      },
+      "put": {
```
Comment (MyronFanQiu) [inline specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json]: Just want to confirm. This is not a long running operation. Is it correct?

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json [hunk 3]
```diff
@@ -2256,6 +2746,98 @@
...
+        },
+        "certVersion": {
+          "description": "The certificate version of key vault.",
+          "type": "string"
+        },
+        "issuer": {
+          "description": "The issuer of certificate.",
+          "type": "string"
+        },
+        "issuedDate": {
```
Comment (MyronFanQiu) [inline specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json]: is there any format for `Date`?

**Review by mmyyrroonn — COMMENTED** [review]
LGTM in general. Just a few questions.

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json (hunk 2 above)
Comment (Incarnation-p-lee) [reply specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json]: yes, it is sync API.

File: specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json (hunk 1 above)
Comment (Incarnation-p-lee) [reply specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json]: updated.

**Comment by Incarnation-p-lee** [comment]
close this PR due to API change.

**Pull request closed without merging**